docs/src/app/(public)/(content)/react/components/popover/page.mdx

```diff
@@ -37,6 +37,16 @@ import { Popover } from '@base-ui-components/react/popover';
 
 ## Examples
 
+### Opening on hover
+
+This example shows how you can configure the popover to open on hover using the `openOnHover` prop.
```

Comment:
I think it's worth adding a callout explaining when this pattern should be used instead of a Tooltip 👀